README.md

```diff
@@ -409,6 +409,19 @@ Similar to the heap profile, you only need to specify the following environment
 
 More information about what environment variables you can configure can be found [here](https://gperftools.github.io/gperftools/cpuprofile.html)
 
+### WebRTC Network Compatibility Checker
+
+This program allows you to understand the network compatibility for WebRTC usage. By providing necessary parameters, it detects NAT behavior and filtering behavior and tells you if you can work with a particular STUN server or not. By default, it uses the STUN server from your environment variable `AWS_DEFAULT_REGION` if present. If not, it uses the STUN server from `us-west-2`.
+#### How to Use
+
+1. **Parameters**:
+   - `-i network-interface-name`: Specify the network interface name.
+   - `-s stun-hostname`: Specify the STUN server hostname.
```

Comment:
May want to mention that `-s` will override the KVS stun server. Also, mention that both parameters are **optional** and what happens if they're not provided.

Also, can we add another sample usage with how to have it use the KVS stun server (including any env variables to export)

Reply:
It already mentions that "By default, it uses the STUN server from your environment variable `AWS_DEFAULT_REGION` if present. If not, it uses the STUN server from `us-west-2`."